### ISSUE TITLE
Spinner/progress, Node v0.12 deprecated util.print "Use console.log instead"

### ISSUE THREAD

**Issue by danielweck** [chriso/cli#63]
My temporary fix:
```javascript
var cli = require('cli');

//cli.native.util.print = console.log; (unnecessary line break)

cli.native.util.print = function(msg) { process.stderr.write(msg) };
```

To reproduce the bug:

https://github.com/chriso/cli/blob/master/examples/spinner.js
```javascript
#!/usr/bin/env node

var cli = require('cli');

cli.spinner('Working..');

setTimeout(function () {
    cli.spinner('Working.. done!', true); //End the spinner
}, 3000);
```

...outputs:

```
util.print: Use console.log instead
Working.. -util.print: Use console.log instead

Working.. \util.print: Use console.log instead

Working.. |util.print: Use console.log instead

Working.. /util.print: Use console.log instead

Working.. done!
```

https://github.com/chriso/cli/blob/master/examples/progress.js
```javascript
#!/usr/bin/env node

var cli = require('cli');

var i = 0, interval = setInterval(function () { 
    cli.progress(++i / 100); 
    if (i === 100) {
        clearInterval(interval);
        cli.ok('Finished!');
    }
}, 50);
```

...outputs:

```
util.print: Use console.log instead
[#                                                                         ] 1%
util.print: Use console.log instead
[##                                                                        ] 3%
util.print: Use console.log instead
[###                                                                       ] 5%
util.print: Use console.log instead
[#####                                                                     ] 7%
util.print: Use console.log instead
...
...
etc.
```

**Comment by danielweck** [comment]
PS: the `cli.output` shorthand is not used anywhere in your code:

https://github.com/chriso/cli/blob/master/cli.js#L67
```
cli.output = cli.native.util.print;
```

**Comment by danielweck** [comment]
Note that on Windows with GitBash / GitShell there are line breaks after each progress message, which spams the terminal with unnecessary text :( 
(even when using `process.stderr.write()`)
That's because `'\u000D'` is not processed correctly in the terminal (see in https://github.com/chriso/cli/blob/master/cli.js )
This works fine in Cygwin though.

**Comment by danielweck** [comment]
Note that https://github.com/tj/node-progress works fine in Windows GitBash and GitShell, because it uses the `process.stderr.clearLine/write/cursorTo()` API. See: https://github.com/mauriciogior/node-progress-1/blob/master/lib/node-progress.js#L132

**Comment by danielweck** [comment]
I filed a separate issue for the terminal line breaking: https://github.com/chriso/cli/issues/64

**Comment by jonniespratley** [comment]
+1 Fix this.

**Comment by chriso** [comment]
Fixed in `0.6.6`.
